tests/EventLoopSpec.hs

```diff
@@ -84,7 +84,7 @@ populateRepository dir doClone =
   let writeFile fname msg  = Prelude.writeFile (dir </> fname) (msg ++ "\n")
       appendFile fname msg = Prelude.appendFile (dir </> fname) (msg ++ "\n")
       git args             = callGit $ ["-C", dir] ++ args
-      gitInit              = void $ git ["init"]
+      gitInit              = void $ git ["init", "--initial-branch", "master"]
```

Comment:
Can you reuse to top level `masterBranch` instead of hard-coding `master` on more locations?

Reply:
sure, i'll remove all hard-coded references ^^